client_src/js/Tour.mjs

```diff
@@ -91,16 +93,17 @@ class Tour {
 			buttons: this.get_buttons()
 		});
 		this.tour.addStep("device-graph-b", {
-			text: "By clicking here, you can see the specification of the device, including its software, sensor models, exact location, more. Click this now.",
+			text: "By clicking here, you can see the specification of the device, including its software, sensor models, exact location, more."
+			+ " Click this now.".bold(),
```

Comment:
Perhaps we could include the `<strong> </strong>` tag explicitly here instead of calling `.bold()`? Same goes for above and below.